### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by magland**

### CODE REVIEW

File: README.md
Comment (rly) [inline README.md:81]: Minor, but could you change the filename of this file to use `.lindi.json`?

**Review by rly — APPROVED** [review]

**Comment by magland** [comment]
@rly I will hold off on merging this until some of the other PRs are merged, because this readme describes some features of those branches.

**Pull request merged**
